src/lib/mergeSchemas.js

```diff
@@ -60,15 +83,28 @@ export async function mergeSchemas() {
     link: convectionLink,
   })
 
+  const gravityTypeDefs = fs.readFileSync(
+    path.join("src/data/gravity.graphql"),
+    "utf8"
+  )
+
+  const gravityLink = createGravityLink()
+
+  const gravitySchema = await makeRemoteExecutableSchema({
+    schema: gravityTypeDefs,
+    link: gravityLink,
+  })
+
   const linkTypeDefs = `
     extend type Submission {
       artist: Artist
     }
   `
 
+  // Add gravity schema first to prefer local MP schema.
+  // Add schemas after localSchema to prefer those over MP.
   const mergedSchema = _mergeSchemas({
-    schemas: [localSchema, convectionSchema, linkTypeDefs],
-    // Prefer others over the local MP schema.
+    schemas: [gravitySchema, localSchema, convectionSchema, linkTypeDefs],
```

Comment:
cc @damassi this is where we actually merge in gravity _first_, so as to keep preferring others further to the right, which we use in `onTypeConflict` below.

Reply:
Makes sense 👍